plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -78,16 +94,7 @@ public void start()
         try {
             this.minio.start();
             this.hiveHadoop.start();
-            AmazonS3 s3Client = AmazonS3ClientBuilder
```

Comment:
We could initialize `s3Client` here itself as a final variable and expose it in a getter API right ? Do we need a new Client for each method ?

Reply:
Unfortunately not. Client is pointing to a Mapped port from Container which may be obtained only when container is started.